config/default/kustomizeconfig.yaml

```diff
@@ -0,0 +1,4 @@
+namePrefix:
+- kind: Deployment
+  name: controller-manager
+  path: spec/template/spec/volumes/secret/secretName
```

Comment:
What is this part doing?

Reply:
See https://github.com/rh-ecosystem-edge/kernel-module-management/pull/508/files#r1178889583